package.json

```diff
@@ -97,6 +97,7 @@
     "lint-staged": "^3.2.6",
     "pre-commit": "^1.2.2",
     "prettier": "^1.3.1",
+    "react-native-cli": "^2.0.1",
```

Comment:
My machine was having problems with running yarn commands that called on `react-native`. The problem was with npm itself, and a workaround listed was to include `react-native-cli` as a dev dependency.